R/survivalFunctions.R

```diff
@@ -222,8 +230,34 @@ PWCsurvOS <- function(t, h01, h02, h12, pw01, pw02, pw12) {
 
   PWCsurvPFS(t, h01, h02, pw01, pw02) +
     exp(-pwA(t, h12, pw12)) *
-    integrateVector(PwcOSInt,
-                    upper = t,
-                    h01 = h01, h02 = h02, h12 = h12, pw01 = pw01, pw02 = pw02, pw12 = pw12
-    )
+      integrateVector(PwcOSInt,
+        upper = t,
+        h01 = h01, h02 = h02, h12 = h12, pw01 = pw01, pw02 = pw02, pw12 = pw12
+      )
+}
+
+
+
+#' Quantile function for OS survival function induced by IDM
```

Comment:
this is new right? maybe better in its own PR? (always best to have one topic per PR)